src/main/java/no/digipost/signature/client/direct/Signature.java

```diff
@@ -26,11 +28,13 @@
     private final String signer;
     private final SignerStatus status;
     private final XAdESReference xAdESReference;
+    private final Date dateTimeForStatus;
```

Comment:
Ok, nå er jeg litt anal her, men kan feltet legges etter `status`? Gjerne grupper med whitespace, f.eks. slik:
```java
private final String signer;
private final XAdESReference xAdESReference;

private final SignerStatus status;
private final Date statusDateTime;
```

I tillegg: hva med på kalle det for `statusDateTime`, `getStatusDateTime()`, etc? Da ser man det lett i IDE-er, og det blir litt sånn visuelt tydeligere at tidspunktet er på en måte "en del av" statusen. 

```java
private final SignerStatus status;
private final Date statusDateTime;
...
this.status = status;
this.statusDateTime = statusDateTime;
...
getStatus()
getStatusDateTime()
```